jellyfin-rpc-cli/src/config.rs

```diff
@@ -44,11 +44,11 @@ pub struct Jellyfin {
 }
 
 /// Contains configuration for Music/Movie display.
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
 pub struct DisplayOptions {
-    /// Display is where you tell the program what should be displayed.
-    pub display: Option<Vec<String>>,
-    /// Separator is what should be between the artist(s) and the `display` options.
-    pub separator: Option<String>,
+    pub state_text: Option<String>,
+    pub details_text: Option<String>,
+    pub image_text: Option<String>,
 }
```

Comment:
This should have doc lines `///` and image_text should probably be large_image_text, there is another one called small_image_text that gets used when you pause content